src/main/resources/static/templates/javascript-producer/src/producer.js

```diff
@@ -0,0 +1,81 @@
+const { Kafka } = require("@confluentinc/kafka-javascript").KafkaJS;
+require("dotenv").config();
+
+const brokers = ["{{ cc_bootstrap_server }}"];
+
+const kafka = new Kafka({
+  kafkaJS: {
+    clientId: "{{ client_id }}",
+    brokers: brokers,
+    connectionTimeout: 10000,
+    ssl: true,
+    sasl: {
+      mechanism: "plain",
+      username: process.env.USERNAME,
+      password: process.env.PASSWORD,
+    },
+  },
+});
+
+const producer = kafka.producer();
+
+// producer should always be connected at app initialization, separately from producing message
+const connectProducer = async () => {
+  await producer.connect();
+  console.log("Connected successfully");
+};
+
+const run = async () => {
+  try {
+    let i = 0;
```

Comment:
We don't use this variable anywhere, consider removing?

Reply:
@Cerchie Mentioning you for visibility.